topological_action_planner/src/topological_action_planner/ed_interface.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function
+
+# TU/e
+import rospy
```

Comment:
This file is going to be replaced with ed_python

Reply:
Hope so. That work is on a  different branch though for now.